docs/source/tutorials/clean_learning/index.rst

```diff
@@ -0,0 +1,11 @@
+CleanLearning Tutorials
+=======================
+
+.. toctree::
+   :maxdepth: 1
+
+   Text Classification <text>
+   Tabular Classification (Numeric/Categorical) <tabular>
+   Image Classification <image>
+   Audio Classification <audio>
```

Comment:
wait you were supposed to move these to the Datalab tutorials:

https://github.com/cleanlab/cleanlab/pull/1066#discussion_r1542124862

Image & Audio tutorial are about Datalab not CleanLearning, right?

Reply:
You're right, silly mistake! I just moved them to the Datalab tutorials in the latest commit